src/procedures/proc_procedures.c

```diff
@@ -0,0 +1,89 @@
+/*
+* Copyright 2018-2020 Redis Labs Ltd. and Contributors
+*
+* This file is available under the Redis Labs Source Available License Agreement
+*/
+
+#include "proc_procedures.h"
+#include "../util/arr.h"
+#include "rax.h"
+#include "procedure.h"
+
+// CALL dbms.procedures()
+
+typedef struct {
+	raxIterator *current_proc; // Current proc
+	SIValue *output; // Array with a maximum of 4 entries: ["name", name, "mode", mode].
```

Comment:
General struct formatting:
1.  take the longest line and write the comment 4 spaces from its end. Align all other lines accordingly.
2. try to sort variables line by length, when possible